packages/babel-preset-env/CONTRIBUTING.md

```diff
@@ -38,19 +38,31 @@ and split it with `/`:
 
 `Object static methods / Object.values`
 
-Find the corresponding module on [core-js](https://github.com/zloirock/core-js/tree/master/modules):
+Find the corresponding module on [`core-js@2`](https://github.com/zloirock/core-js/tree/v2/modules):
 
-`es7.object.values.js`
+`es.object.values.js`
```

Comment:
Doesn't `core-js@2` still use `esNUMBER` prefixes?

Reply:
Hmm... Seems I forgot to revert it.